<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateDialog</class>
 <widget class="QDialog" name="UpdateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update Item</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_slot">
     <property name="title">
      <string>Slot</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_slot_warehouse">
        <property name="text">
         <string>Warehouse</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_warehouse">
        <property name="styleSheet">
         <string notr="true">background-color: lightgrey;
border-color: darkgrey;</string>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="duplicatesEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_slot_slot">
        <property name="text">
         <string>Slot</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_slot_slot</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_slot_slot">
        <property name="styleSheet">
         <string notr="true">background-color: lightgrey;
border-color: darkgrey;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_product">
     <property name="title">
      <string>Product</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_product_sku">
        <property name="text">
         <string>SKU</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_product_sku</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_product_sku">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_product_lookup">
        <property name="text">
         <string>Lookup</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_product_description">
        <property name="text">
         <string>Description</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_product_description</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_product_description">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_date_received">
        <property name="text">
         <string>Received</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_date_received</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_date_received"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_date_packed">
        <property name="text">
         <string>Packed</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_date_packed</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEdit_date_packed"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_date_expiration">
        <property name="text">
         <string>Expiration</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_date_expiration</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_date_expiration"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_quantity">
        <property name="text">
         <string>Quantity</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_quantity</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEdit_quantity"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_details_notes">
        <property name="text">
         <string>Notes</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_details_notes</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEdit_details_notes"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_clear">
       <property name="text">
        <string>Undo Changes</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_slot_slot</tabstop>
  <tabstop>lineEdit_product_sku</tabstop>
  <tabstop>pushButton_product_lookup</tabstop>
  <tabstop>lineEdit_product_description</tabstop>
  <tabstop>lineEdit_date_received</tabstop>
  <tabstop>lineEdit_date_packed</tabstop>
  <tabstop>lineEdit_date_expiration</tabstop>
  <tabstop>lineEdit_quantity</tabstop>
  <tabstop>lineEdit_details_notes</tabstop>
  <tabstop>pushButton_clear</tabstop>
  <tabstop>pushButton_ok</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_quantity</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton_ok</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>676</x>
     <y>523</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
